### PR TITLE
Don't include clients with absurd search counts in aggregation

### PR DESCRIPTION
**Pull request by wlach**

### CODE REVIEW

**Comment by wlach** [comment]
This hopefully fixes: https://bugzilla.mozilla.org/show_bug.cgi?id=1412040

File: mozetl/search/dashboard.py [hunk 1]
```diff
@@ -70,6 +70,7 @@ def explode_search_counts(main_summary):
         .withColumn(exploded_col_name, explode(col('search_counts')))
         .withColumn('addon_version', udf_get_search_addon_version('active_addons'))
         .select(['*'] + search_fields)
+        .filter('single_search_count.count < 1000')
```
Comment (harterrt) [inline mozetl/search/dashboard.py]: Can we bump this up to 10k to be safe? IIRC, the outrageous values tend to be hundreds of millions of searches per day.

Comment (wlach) [reply mozetl/search/dashboard.py]: Makes sense, I'll do that

Comment (fbertsch) [reply mozetl/search/dashboard.py]: Drive-by comment: move this to a well-named constant which can also be used in the tests.

Comment (wlach) [reply mozetl/search/dashboard.py]: Also a good idea, done

**Pull request merged**